ui/app/src/Main.elm

```diff
@@ -8,7 +8,7 @@ import Types
         ( Route(..)
         , Msg
             ( NavigateToSilenceList
-            , NavigateToSilence
+            , NavigateToSilenceView
```

Comment:
Because I renamed `Silence` to `SilenceView`, which represents a page. I didn't want to confuse it with a Silence type.